build/ci.mk

```diff
@@ -48,8 +49,29 @@ ifdef TAG
 			-w /go/src/github.com/newrelic/nri-$(INTEGRATION) \
 			-e INTEGRATION=$(INTEGRATION) \
 			-e TAG \
-			$(BUILDER_TAG) make release/build
+			$(BUILDER_IMAGE) make release/build
```

Comment:
also ci/build is used only in tests, I think we should test in the same way we release, I would remove this target to avoid confusion

Reply:
the only reason to keep it would be if in the close future we do docker in docker or if we change the way we build the image. But I do not think we are going to do it in the close future

Reply:
> the only reason to keep it would be if in the close future we do docker in docker or if we change the way we build the image. But I do not think we are going to do it in the close future

On the contrary my assumption was that we'll be changing the way we build images in near future. Moreover, with go 1.24, we might not need to build FIPS images in docker and could just build on the runner. But that is still 3 months away. IMO, we should make `nri-prometheus` similar to other OHI release flows. Right now it is very confusing and messy. But would be difficult to block FIPS work because of this.

Reply:
what I meant is -> why do we have `ci/build` if in other parts we use directly `make release/build`